policy/modules/contrib/virt.te

```diff
@@ -14,13 +14,19 @@ attribute virsh_transition_domain;
 attribute virt_ptynode;
 attribute virt_system_domain;
 attribute virt_domain;
+attribute virt_driver_domain;
+attribute virt_driver_executable;
+attribute virt_driver_var_run;
 attribute virt_image_type;
 attribute virt_tmpfs_type;
 attribute svirt_file_type;
 attribute virt_file_type;
 attribute sandbox_net_domain;
 attribute sandbox_caps_domain;
 
+permissive virtxend_t;
+permissive virtvzd_t;
```

Comment:
There could also be a code comment explaining why these are permissive for now.

Reply:
There is also a possibility to add these domains for Fedora here: https://src.fedoraproject.org/rpms/selinux-policy/blob/master/f/permissivedomains.cil 

and we can keep policy sources without permissive domain.

Reply:
I don't think that would make much sense. If the policy is absent, it simply wouldn't work without the `permissive`s. And it's not good to have this information in a different repo - it's *here* that the policy needs to be added in the future, so the permissive declarations should also be here. Then they can also be removed atomically in the same commit that adds the missing policy.

Reply:
No problem, as you say. :) In past, we kept permissive domains there. Also, if there is no SELinux policy for these types, what about unconfined_domain() macro? Using this macro we avoid lots of SELinux policy denials and confused users. WDYT?

Reply:
That might indeed be a better approach, but I don't know enough about either approach to judge :)

Reply:
I think the main question is: do we want to have users receive AVCs and report them or not?

IMHO in this stage of development should rather be permissive.

Reply:
I would prefer to have AVC's reported from Libvirt team, not users. For each driver I had many AVC messages and it might be confusing for users. Maybe it will be better, to use unconfined_domain macro until the AVC messages are reproduced internally and policy for these drivers is made.

Reply:
Agreed. I'd use the unconfined_domain here and then permissive domains in the policy libvirt-selinux will ship (this version will be tested by libvirt team). This way users can start using the new policy (split-daemon configuration will be used by default soon), we have time to polish the policy based on libvirt testing, and any issues still present in the final policy can be reported by users.